go-controller/pkg/node/controllers/egressip/egressip.go

```diff
@@ -1321,7 +1321,10 @@ func overwriteRoutesTableID(routes []netlink.Route, tableID int) {
 }
 
 func getRouteTableID(ifIndex int) int {
-	return ifIndex + routingTableIDStart
+	if !util.IsNetworkSegmentationSupportEnabled() {
+		return ifIndex + routingTableIDStart
+	}
+	return ifIndex + int(ovnconfig.UserDefinedNetworks.VRFTableBase) + int(ovnconfig.UserDefinedNetworks.MaxNetworks)
```

Comment:
not sure if I follow the math here. the vrf base is 256, + 30K, ? why is that? isn't it just supposed to be 30K+ifIndex?